ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master-presubmits.yaml

```diff
@@ -0,0 +1,156 @@
+presubmits:
```

Comment:
@dewan-ahmed Please run `make jobs` and commit the changes.

NOTE: Whenever you you do any changes in the config file, you need to generate it job file by running `make jobs`